process.py

```diff
@@ -399,10 +411,15 @@ def main():
             db.session.query(table).delete()
             db.session.commit()
 
-    for directory in glob.glob("{0}/*/*".format(args.path)):
-        parent_directory = os.path.basename(os.path.dirname(os.path.join(os.pardir, directory)))
-        provider_code = args.provider_code if args.provider_code else get_provider_code(parent_directory)
-        import_to_datastore(directory, provider_code, args.batch_size)
+    if args.specific_folder:
+         parent_directory = os.path.basename(os.path.dirname(os.path.join(os.pardir, dirname)))
+         provider_code = args.provider_code if args.provider_code else get_provider_code(parent_directory)
+         import_to_datastore(dirname, provider_code, args.batch_size)
+    else:
+        for directory in glob.glob("{0}/*/*".format(args.path)):
```

Comment:
I can't because inside the for it needs to get the directory itself. If I give the path to one directory - It doesn't have any directories inside so it won't enter the for loop.